flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
+}
```

Comment:
why restrict to x86_64 only ?

Reply:
Couldn't test it on Arm or Aarch64, not sure it'll work.
Should I unrestrict it?

Reply:
you can remove it so that we can check if it builds. It's a chicken and egg issue, if there is no build present no one will be able to test it.